debian/rules

```diff
@@ -1,12 +1,61 @@
 #!/usr/bin/make -f
+#export DH_VERBOSE=1
 
-export PYBUILD_NAME=python3-ev3dev
+export PYBUILD_NAME=ev3dev2
```

Comment:
I am not 100% sure, but I believe this name was being ignored before because we had only one binary package. Once I added the second one this started producing files under `debian/python3-python3-ev3dev/`, which is wrong both because of the duplicate `python3-` and because the package name is supposed to have a `2` at the end. So I replaced the name here and things seem to go to the right place.